lib/core/macros.nim

```diff
@@ -546,6 +547,9 @@ proc getAst*(macroOrTemplate: untyped): NimNode {.magic: "ExpandToAst", noSideEf
   ##     var ast = getAst(BarTemplate())
 
 proc quote*(bl: typed, op = "``"): NimNode {.magic: "QuoteAst", noSideEffect.} =
+  ## .. warning:: `quote` comes with many caveats, use `genasts <genasts.html>`_
+  ##   instead.
```

Comment:
We need to watch out that in two months genAst doesn't grow the same warning. Features usually work well enough for the original designer, it's the **other users** who complain and find design problems.

Reply:
done, i changed the wording to a "see also" reference, for now.